src/styles/index.scss

```diff
@@ -1,3 +1,124 @@
+* {
+  font-size: 14px;
+  text-decoration: none;
+  box-sizing: border-box;
+}
+@import 'vars';
+
+html{
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
+
+.card {
+  position: relative;
+  width: 200px;
+  height: 406px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  .card__img {
+    width: 100%;
+    max-width: 160px;
+    height: 134px;
+    display: block;
+    margin: auto;
+  }
+
+  .card__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $color-dark-blue;
+    margin-top: 40px;
+  }
+
+  .card__prod-code {
+    margin-top: 4px;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    color: $color-dark-grey;
+  }
+
+  .card__block {
+    margin-top: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+
+    .stars {
+      display: flex;
+
+      .stars__star {
+        width: 16px;
+        height: 16px;
+        background-position: center;
+        margin-right: 4px;
+
+        background-repeat: no-repeat;
+        background-image: url(../images/star-active.svg);
+      }
+
+      .stars__star:nth-child(5) {
+        background-image: url('../images/star.svg');
+      }
+    }
+    .card-review {
+      font-size: 10px;
+      line-height: 14px;
+      color: $color-dark-blue;
+    }
+  }
+
+  .card__price {
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    .card__txt {
+      font-size: 12px;
+      line-height: 18px;
+      color: $color-dark-grey;
+    }
+    .card__money {
+      font-size: 16px;
+      line-height: 18px;
+      color: $color-dark-blue;
+      font-weight: 700;
+    }
+  }
+
+  .btn-inner {
```

Comment:
Not fixed same
![2024-04-11_22h40_33](https://github.com/mate-academy/layout_product-cards/assets/61904995/c6adf75a-13bd-4047-9179-33074109c9f2)

Reply:
All BEM tests were successful. What I have do to? Could you explain?